x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{3_4,3_5,3_6} )
```

Comment:
could you check if it may support `python3_7`?

Reply:
I've already tried it. Repoman produces too many errors with python3_7.
I rather avoid use python3_7 until it will be supported in packages in DEPEND.